site/src/playground/examples/horizon.md

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Can we omit the horizon example for the time-being? I really like the code / concept and want to see it in the core of d3fc. I'd like to keep our website, and where we place examples, as simple as possible.